tools/ci/travis/travis-ci.sh

```diff
@@ -80,15 +80,18 @@ critical errors that must be understood and fixed before your pull
 request can be merged.
 ------------------------------------------------------------------------
 EOF
+
+  configure_flags="\
+    --prefix=$PREFIX \
+    --enable-flambda-invariants \
+    $CONFIG_ARG"
   case $XARCH in
   x64)
-    ./configure --prefix $PREFIX -with-debug-runtime \
-      -with-instrumented-runtime -with-flambda-invariants $CONFIG_ARG
+    ./configure $configure_flags
```

Comment:
I guess the debug runtime is on by default, but you removed `-with-instrumented-runtime` ??